src/pages/app-development/app-builder-partnership-essentials.md

```diff
@@ -1,53 +1,90 @@
 
-- Will you publish an application as an individual or on behalf of a company?
-- Whose name will you publish the application under?
-- What is your area of expertise (for example, payments, checkout, marketing, social, or tax)?
-- Who will own and maintain the application?
-- Are you committed to building and publishing the application?
-- How much time or resources are you able to devote to this application development?
+Through the **Technology Partner Program (or TPP)**, you will receive access to the tools and resources you will need. Adobe is committed to helping you navigate challenges and complete your apps successfully.
+
+Together, we can create solutions that deliver real value to you, your customers, and the broader Adobe ecosystem.
 
 ## Requirements to access App Builder
 
+Access to App Builder requires the following conditions:
+
 - **Membership requirements** - To build and publish an application, you must have a Silver tier membership or higher, which gives you sandbox access and the possibility to publish on the Adobe Exchange Marketplace. Start with your membership and reach out to Adobe to share your objectives on app building and start a collaboration between your team and ours. We commit to our partners and will support you if you have questions or need help during the development stage.
 
-- **Account requirements** (Technology Partner Program) - The next step is to [create a Technology Partner Program (TPP) account](https://partners.adobe.com/technologyprogram/experiencecloud/registration.html), which is required to access App Builder, the Adobe Commerce as a Cloud Service sandbox and the App Builder sandbox, whether you are a Solution Implementer (SI) or Tech partner.
+- **Account requirements** - A [TPP account](https://partners.adobe.com/technologyprogram/experiencecloud/registration.html) is required to access all sandboxes (App Builder, the Adobe Commerce as a Cloud Service SaaS sandbox, and the PaaS sandbox), whether you are a Solution Implementer (SI) or an Independent Software Vendor.
 
 <InlineAlert variant="info" slots="text"/>
 
-If you are a Solution Partner Program (SPP) partner, you will need to create a Technology Partner Program (TPP) account to access these environments.
+If you are a **Solution Partner Program (SPP)** partner, you will need to create a TPP account to access these environments.
+
+### How to set up your account
+
+#### Step 1: Request a TPP account
+
+[Create a Silver-level (or higher) TPP account](https://partners.adobe.com/technologyprogram/experiencecloud/registration.html) by using a personal corporate email address, and a valid company website. Follow the steps described [here](https://partners.adobe.com/technologyprogram/experiencecloud/knowledgebase/a96d596f933f8214fbe77b847aba101c.html) to Register for the Program.
+
+   ![Register to the TPP program](../_images/register_tpp_adobe.png)
```

Comment:
I'm looking at a local preview build, and I think you should delete this screenshot. It's too big and too text heavy to be useful.

Reply:
I have to discuss this today with the PM